### PR TITLE
[WIP] Tries to integrate TagsBundle in Hybrid Platform UI v2

### PR DESCRIPTION
**Pull request by dpobel**
This is just to demo how this could be made during the eZ Conference :)

**Don't merge as is** :)

### CODE REVIEW

File: Resources/config/routing.yml [hunk 1]
```diff
@@ -10,4 +10,4 @@ netgen_tags_admin_redirects:
 
 netgen_tags_admin:
     resource: "routing/admin.yml"
-    prefix: /tags/admin
+    prefix: /pjax/tags/admin
```
Comment (emodric) [inline Resources/config/routing.yml]: Is this absolutely necessary? :)

Tags admin UI can work as as standalone app, so I'd prefer if I don't have to use `/pjax` prefix for that.

File: PlatformUI/EventListener/PlatformUIListener.php [hunk 2]
```diff
@@ -19,10 +19,6 @@ protected function isPlatformUIRequest(Request $request)
             return false;
         }
 
-        if (!$request->headers->has('X-PJAX') || !$request->isXmlHttpRequest()) {
-            return false;
-        }
-
-        return true;
+        return $request->headers->has('X-AJAX-Update') || $request->isXmlHttpRequest();
```
Comment (emodric) [inline PlatformUI/EventListener/PlatformUIListener.php]: Is this right?

Not every AJAX request is Platform UI 2.x request :)

Will this work for Platform UI 1.x?

File: Resources/config/routing.yml (hunk 1 above)
Comment (dpobel) [inline Resources/config/routing.yml]: I know, I mentioned that during the conference, this is something we'll improve, but for now that the only way. Hence why, the *WIP* in the title :)

File: PlatformUI/EventListener/PlatformUIListener.php (hunk 2 above)
Comment (dpobel) [inline PlatformUI/EventListener/PlatformUIListener.php]: well, same answer as in the other comment. Actually, I should have titled this PR *POC* :)

**Comment by emodric** [comment]
Closing as obsolete. Platform UI is deprecated and there's now a preliminary integration into eZ Platform Admin UI: https://github.com/netgen/TagsBundle/commit/f9b17dbec735e99964b0c2d0bc332f349bd213f9

**Pull request closed without merging**